web_src/js/utils/dom.js

```diff
@@ -0,0 +1,65 @@
+function getComputedStyleProperty(el, prop) {
+  const cs = el ? window.getComputedStyle(el) : null;
+  return cs ? cs[prop] : null;
+}
+
+function isShown(el) {
+  return getComputedStyleProperty(el, 'display') !== 'none';
+}
```

Comment:
`classList.contains`?

Reply:
No, it must get the computed styles to catch bugs. And it only needed when RunMode=dev

There is no affect for production.